File: step5/8958.py
T = int(input())
for i in range(T):
    res = input().split("X")
    print(sum(sum(range(len(elem) + 1)) for elem in res))
    # res = input()
    # score = 0
    # total = 0
    # for j in range(len(res)):
    #     if res[j] == "X":
    #         score = 0
    #     else:
    #         score += 1
    #     total += score
    # print(total)

File: step5/8958.c
#include <stdio.h>
#include <string.h>

int main(void)
{
    char res[80];
    int T, score = 0, total = 0;
    scanf("%d", &T);
    for (int i = 0; i < T; i++) {
        scanf("%s", res);
        score = 0, total = 0;
        for (int j = 0; j < strlen(res); j++) {
            if (res[j] == 'X')
                score = 0;
            else
                score++;
            total += score;
        }
        printf("%d\n", total);
    }
    return 0;
}